Dockerfile

```diff
@@ -8,7 +8,8 @@ RUN mvn dependency:go-offline
 COPY . /isaac-api
 RUN mvn package -Dmaven.test.skip=true -Dsegue.version=$BUILD_VERSION $MVN_PACKAGE_PARAM
 
-FROM jetty:11.0.20-jdk11-eclipse-temurin
+
+FROM jetty:11.0.20-jdk17-eclipse-temurin
```

Comment:
I'm not sure it actually matters but you seem to have shuffled some of the empty lines around here?